test/docker_test.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+set -e
+
+test_dir=$(pwd -P)
+
+mkdir -p example
+cd example
+if [ ! -f Homo_sapiens.GRCh37.75.dna.chromosome.22.fa ]
+then
+	if [ ! -f Homo_sapiens.GRCh37.75.dna.chromosome.22.fa.gz ]
+	then
+		docker run -v ${test_dir}:/mnt -u $UID --memory 30G  msahraeian/neusomatic:0.1.1 /bin/bash -c \
```

Comment:
Have you already pushed to the Docker hub?

Reply:
yes